<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-UASL38CC" failures="2" tests="5" name="ContactManager" time="79.387" errors="0" timestamp="29 Jun 2020 14:22:25 GMT">
  <testcase classname="com.qa.contacts.tests.ContactsHomePageTest" name="verifyContactHeaderTest" time="0.274"/>
  <testcase classname="com.qa.contacts.tests.BackupRestoreTest" name="verifyRestoreContactsTextTest" time="5.004">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.contacts.tests.BackupRestoreTest.verifyRestoreContactsTextTest(BackupRestoreTest.java:43)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- verifyRestoreContactsTextTest -->
  <testcase classname="com.qa.contacts.tests.ContactsHomePageTest" name="verifySetPasswordTest" time="0.15"/>
  <testcase classname="com.qa.contacts.tests.BackupRestoreTest" name="verifyContactClickTest" time="0.004">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.contacts.tests.BackupRestoreTest.verifyContactClickTest(BackupRestoreTest.java:55)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- verifyContactClickTest -->
  <testcase classname="com.qa.contacts.tests.ContactsHomePageTest" name="goToBackUpRestoreTest" time="0.836"/>
</testsuite> <!-- ContactManager -->
